Source/Diagnostics/WarpXIO.cpp

```diff
@@ -13,11 +13,11 @@
 #endif
 #include "FieldIO.H"
 #include "Particles/MultiParticleContainer.H"
-#include "Utils/CoarsenIO.H"
 #include "Utils/TextMsg.H"
 #include "Utils/WarpXProfilerWrapper.H"
 #include "WarpX.H"
 
+#include <ablastr/coarsen/sample.H>
```

Comment:
Do you think we can remove this, since it doesn't seem to be used in the file?

Reply:
Thanks, true!